ibm-randori/connector.py

```diff
@@ -0,0 +1,26 @@
+"""
+Copyright start
+MIT License
+Copyright (c) 2024 Fortinet Inc
+Copyright end
+"""
+
+from connectors.core.connector import Connector, get_logger, ConnectorError
+from .operations import operations, check_health
+
+logger = get_logger('ibm_randori')
+
+
+class IBMRandori(Connector):
+    def execute(self, config, operation, params, **kwargs):
+        logger.info('execute [{}]'.format(operation))
+        try:
+            operation = operations.get(operation)
+            return operation(config, params)
+        except Exception as err:
+            logger.exception("An exception occurred [{}]".format(err))
+            raise ConnectorError("An exception occurred [{}]".format(err))
+
+    def check_health(self, config):
+        logger.info('starting health check')
```

Comment:
Both operations.py and connector.py have same function name and is used inside connector.py, Please change the operations.py check_health to _check_health or any good method name for better readability.